spacy/tokens/span.pyi

```diff
@@ -51,7 +53,12 @@ class Span:
         kb_id: Union[str, int] = ...,
         span_id: Union[str, int] = ...,
     ) -> None: ...
-    def __richcmp__(self, other: Span, op: int) -> bool: ...
```

Comment:
Does it make sense to remove this entirely?

Reply:
```python
from spacy.tokens import Span
Span.__richcmp__
```

Fails with an `AttributeError: 'spacy.tokens.span.Span' object has no attribute '__richcmp__'`.

I think Cython mangles the method name in some way while creating the six operators.